### ISSUE TITLE
Unexpected am/pm results with object input 

### ISSUE THREAD

**Issue by emoulson** [spencermountain/spacetime#82]
The `ampm` parameter of the object date input seems to default to `pm` when it's passed `null` or an empty string without a timezone. All the other params that I've tested seem to fallback to `.now()` values, but 'ampm' always defaults to `pm`, which seems a bit off, especially when `null` or a blank string are passed (I would expect `am`).

Here's what I tested:
```javascript
spacetime({month:'12', day:'25', hour:'6', minute:'24', ampm:null}).format('nice')
/* and */
spacetime({month:'12', day:'25', hour:'6', minute:'24', ampm:''}).format('nice')

/* return "Dec 25th, 6:24pm" */
```

This is really obvious if you try to pass a 24h time:
```javascript
spacetime({month:'12', day:'25', hour:'15', minute:'24', ampm:null}).format('nice')
/* and */
spacetime({month:'12', day:'25', hour:'15', minute:'24', ampm:''}).format('nice')

/* return "Dec 26th, 3:24am" */
```

If you add a timezone, or remove the `ampm` param altogether, it works as expected (defaults to am):
```javascript
spacetime({month:'12', day:'25', hour:'15', minute:'24', ampm:null}, 'America/New_York').format('nice')
/* and */
spacetime({month:'12', day:'25', hour:'15', minute:'24'}).format('nice')

/* return "Dec 25th, 3:24pm" */
```

**Comment by spencermountain** [comment]
good find Evan, i'm sure this is a lazy falsy-check on my part.
Will get it in the new year.
cheers

**Comment by spencermountain** [comment]
fixed in `5.1.1`
